vss-accessor/src/vss_error.rs

```diff
@@ -0,0 +1,78 @@
+use crate::vss::{ErrorCode, ErrorResponse};
+use prost::bytes::Bytes;
+use prost::{DecodeError, Message};
+use reqwest::StatusCode;
+use std::error::Error;
+use std::fmt::{Display, Formatter};
+
+/// When there is an error while writing to vss-storage, the response contains a relevant error code.
+/// This enum is direct mapping of vss-server error codes. Refer docs for [`ErrorResponse`] for more
+/// information regarding each error code and corresponding use-cases.
+#[derive(Debug)]
+pub enum VssError {
+	InvalidRequestError(String),
+	ConflictError(String),
+	InternalServerError(String),
+	InternalError(String),
```

Comment:
Should this be `UnknownError`? Otherwise, hard to distinguish this from `InternalServerError`.

Reply:
Can't it arise if the client library is out-of-sync with the server? Unknown seems appropriate there.